instrumentation/ktor/ktor-1.0/library/src/main/kotlin/io/opentelemetry/instrumentation/ktor/v1_0/KtorServerTracing.kt

```diff
@@ -52,6 +56,12 @@ class KtorServerTracing private constructor(
       this.statusExtractor = extractor
     }
 
+    fun setSpanKindExtractor(
```

Comment:
I have not written any tests for this method, as there were no unit tests for the existing setStatusExtractor or addAttributeExtractor, is that ok?

Reply:
Thank you for your detailed explanation, I added test cases in https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/8255/commits/959ee02294c2244fa56a078ecd9d05c9292954c1.